sealtk/core/KwiverVideoSource.cpp

```diff
@@ -24,6 +25,23 @@ namespace sealtk
 namespace core
 {
 
+namespace // anonymous
+{
+
+// ----------------------------------------------------------------------------
+kv::path_t getImageName(kv::metadata_vector const& mdv)
+{
+  for (auto const& mdp : mdv | kvr::valid)
+  {
+    if ( auto* const mdi = mdp->get( kwiver::vital::VITAL_META_IMAGE_URI ) )
```

Comment:
This is blocked by Kitware/kwiver#782. I will hold off on this PR until that is merged.

Reply:
Right, good catch, good plan. (On which note, @linus-sherrill, ping?)